shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  isCurrentRoute: function (routeName) {
+    return Router.current().route.getName() == routeName;
```

Comment:
FWIW (not really an excuse) I am copy-pasta-ing from `topbar.js`. I am happy to clean up both in the same commit.